packages/fetchye/src/computeKey.js

```diff
@@ -18,14 +18,14 @@ import computeHash from 'object-hash';
 import mapHeaderNamesToLowerCase from './mapHeaderNamesToLowerCase';
 
 export const computeKey = (key, options) => {
-  const { headers, ...restOfOptions } = options;
+  const { headers, mapKeyToCacheKey, ...restOfOptions } = options;
```

Comment:
This is teetering on the edge of a breaking change. Just a watch item for now though wrt future changes

Reply:
Yah I considered it. but there is no reason to pass 'mapKeyToCacheKey' already, since it will be ineffectual. 

Since it is only breaking for people who already pass this key, I didn't mark it as breaking.